### ISSUE TITLE
[BUG] api not connecting to Oobabooga

### ISSUE THREAD

**Issue by elfgoose** [Cohee1207/SillyTavern#96]
**Describe the bug**
When I try to connect to Pygmalion running on Oogabooba, it doesn't work.  It fails to connect and in the Ooga window, I just get repeated messages saying 
127.0.0.1 - - [18/Apr/2023 01:19:55] code 404, message Not Found
127.0.0.1 - - [18/Apr/2023 01:19:55] "GET /api HTTP/1.1" 404 -

It was working fine and is still working for base TavernAI

**To Reproduce**
Happens every time I try to connect to server

**Expected behavior**
Normally it just connects and then works

**Desktop (please complete the following information):**
 - OS/Device: Windows 10
 - Environment: Local
 - Browser Firefox (same in chrome)
 - Generation API: Oobabooga
 - Branch main
 - Model Pygmalion mayaeary_pygmalion-6b-4bit-128g

**Comment by elfgoose** [comment]
Yes, exactly.  And it used to work in sillytavern like that too, but doesn't now

**Comment by shadowlinyf** [comment]
which url should I use? /api or the webui url?

**Comment by shadowlinyf** [comment]
> 127.0.0.1:5000/api if you're localhosting

Ok,Thank you, I have the same problem as op. So I need to use dev branch,right?

**Comment by shadowlinyf** [comment]
> > So I need to use dev branch,right?
> 
> Yes, I tested today on dev with recent fixes and it worked for me. I was able to get a reply from oobabooga running in Kobold mode

Should I choose oobabooga or Kobold in sillytavern?

**Comment by shadowlinyf** [comment]
Can I use oobabooga mode?

**Comment by shadowlinyf** [comment]
> Yes, you can. Just use another port and no /api at the end (by default it's 7860). The ticket was about specifically the Kobold compatibility extension.

I see. Thanks
